pennylane/operation.py

```diff
@@ -300,48 +395,68 @@ def hash(self):
         """int: returns an integer hash uniquely representing the operator"""
         return hash((str(self.name), tuple(self.wires.tolist()), _process_data(self)))
 
-    @classmethod
-    def _matrix(cls, *params):
-        """Matrix representation of the operator
-        in the computational basis.
+    # pylint:disable=unused-argument
+    @staticmethod
+    def compute_matrix(*params, **hyperparams):
```

Comment:
I suppose it makes sense that this doesn't take `wires` right

Reply:
Yes, and where a base matrix needs info like the number of wires, we just overwrite the signature!

Reply:
One or two operations _do_ need wires. We have two options: a) pass it here by default (although it will be unused in most cases), or b) handle these cases separately. I opted for b) because I think we have to provide for deviations like this anyways.